docs/sphinx-docs/build_sphinx.py

```diff
@@ -205,6 +205,7 @@ def retrieve_user_docs():
                     fromhere=os.path.join(SASMODELS_SOURCE_PROLOG,files)
                     tohere=os.path.join(SASMODELS_DEST_PROLOG,files)
                     shutil.copy(fromhere,tohere)
+    else: print "no source directory",SASMODELS_SOURCE_PROLOG, "was found"
```

Comment:
print should be a function. Missing a space after the comma and before SASMODELS_SOURCE_PROLOG